FGEGraphics/UISystem/UIElement.cs

```diff
@@ -310,71 +310,82 @@ public virtual void Tick(double delta)
     {
     }
 
-    /// <summary>Performs a tick on this element's children.</summary>
+    /// <summary>Recursively ticks this element's children.</summary>
     /// <param name="delta">The time since the last tick.</param>
     public virtual void TickChildren(double delta)
     {
-        int mX = (int)Window.MouseX; // TODO: Propagate float support.
-        int mY = (int)Window.MouseY;
-        bool mDown = Window.CurrentMouse.IsButtonDown(MouseButton.Left);
-        foreach (UIElement element in ElementInternal.Children)
+        foreach (UIElement child in ElementInternal.Children)
         {
-            if (!element.IsValid)
+            if (child.IsValid)
             {
-                continue;
-            }
-            if (element.Contains(mX, mY))
+                child.FullTick(delta);
```

Comment:
With the change of separating out 'TickInteraction' -- what actually needs the forward-Tick here?

Or in other words: did this actually need to be separated to two tick methods, instead of just reversing the call of tick?

Reply:
Really just for organizational purposes. `TickInteraction` has nothing to with parent/child relationships anymore. You can't put it in `FullTick` either because that's recursive